modules/doc/content/help/inl/hpc_ondemand.md

```diff
@@ -42,8 +41,7 @@ The following represents some of the more useful features, and is not an exhaust
 
   - Launches a new browser tab granting an interactive shell prompt to the [!ac](HPC) machine
    selected. The interactive shell will be one of the more useful tools when needing direct access
-   to things like: launching PBS jobs manually, building MOOSE, obtaining a
-   [Civet-like testing environment](faq/apptainer.md) and more.
+   to things like: launching PBS jobs manually, building MOOSE, running tests, and more.
```

Comment:
```suggestion
   to things like: launching PBS/Slurm jobs manually, building MOOSE, running tests, and more.
```

Since Slurm is now a thing at INL on the newest machine. 😄